.github/workflows/backend-tests.yml

```diff
@@ -1,6 +1,10 @@
 name: Backend (HMS API) Tests
 
-on: pull_request
+on:
+  pull_request:
+  push:
+    branches:
+      - '*'
```

Comment:
so I was attempting to avoid it doing too many runs in case you did a lot of pushes, but I doubt we would exceed some limit in GitHub to be honest